src/controls/ControlManager.ts

```diff
@@ -138,6 +138,9 @@ export abstract class ControlManager implements IControlManager {
      */
     constructor(props?: ControlManagerProps) {
         this.rawProps = props;
+        if (props !== undefined && props.services !== undefined) {
+            ControlServices.setDefaults(props.services);
```

Comment:
I don't think this is right.. Initializing a controlManager with custom services should not stomp on the registered defaults.  If developers want to set the defaults they should do it explicitly.

(This could cause some difficult bugs.)

Reply:
Got it, updated!